docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,506 @@
+
+```
+constructor(em) {
+    ...
+    this.org = null;
+}
+activate() {
+    ...
+    this.evt.on('lcSuspend', this.returnToMain);
+    ...
+}
+deactivate() {
+    ...
+    this.evt.off('lcSuspend', this.returnToMain);
+    ...
+}
+myPrelaunch(winId, orig) {
+    this.org = orig;
+    ...
+}
+resume(winId) {
+    ...
+}
+returnToMain(winId) {
+    addonAPI.window.loadURL(winId, this.org);
+}
+```
+
+To test the implementation, follow these steps:
+
+1. In the Web application, press the **Home** key to enter the suspend state.
+2. Long-press the **Home** key and select **Basic UI** to enter the resume state. You can see the main page of Basic UI Web application:
+
+    ![Basic UI Web Application](media/basic_app_running_1_mw.png)
+
+#### Using Back key
+
+When you press the **Back** key in the **hello.html** page, nothing happens. It is usually expected to return to a previous page from the current page after pressing the **Back** key.
+To implement it, modify **hello.html** as shown in the following code:
+
+```
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+    <script type="text/javascript">
+        window.onload = function() {
+            document.addEventListener("tizenhwkey", function(e) {
+                if (e.keyName == "back") try {
+                    // your action
+                } catch (t) {}
+            });
+        }
+    </script>
+</head>
+```
+
+You can add your code in the `try` block to enable the **Back** key functionality in the **hello.html** page. In this case, it must return to the main page of the Web application. The preceding code is working on the renderer process. However, the API loadURL that you used earlier to change the page of the current window is working at the browser process. The browser process must know when the **Back** key is pressed.
+
+There is a set of APIs for the messaging system between the two processes in the Addon APIs. You must specify the name of the message as you want. In this example, you can use `backkey` as the name. Then, in **hello.html**, it must send a message `backkey` to the browser process when the **Back** key is pressed. You can replace your action using the following code:
+
+```
+require('addonapi').messaging.send('backkey');
+```
+
+Additionally, you need to add some code to handle the message in `myPrelaunch()` of **tmplDefault.js** at browser process:
+
+```
+myPrelaunch(winId, orig) {
+    ...
+    var this_ = this;
+    addonAPI.messaging.on('backkey', (event, msg) => {
+        addonAPI.window.loadURL(winId, this_.org);
+    });
+}
+```
+Now, when you test the newly built package after modifying the **hello.html** file, **Back** key in **hello.html** will take you to the main page of the Web application.
```

Comment:
Please add "the" before "**Back** key".
Apply the same in line number 438 also.

Reply:
done